tests/tensorflow/test_tensorflow2_metric_value_convertion_utils.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+
+import mlflow
+from mlflow import tracking
+from mlflow.tracking.fluent import start_run
+from mlflow.exceptions import MlflowException, INVALID_PARAMETER_VALUE, ErrorCode
+from mlflow.tracking.metric_value_conversion_utils import convert_metric_value_to_float_if_possible
+
+import tensorflow as tf
+
+
+def test_reraised_value_errors():
+    multi_item_tf_tensor = tf.random.uniform([2, 2])
+
+    with pytest.raises(
+        MlflowException, match=r"Expected metric value to contain a single element"
+    ) as e:
+        convert_metric_value_to_float_if_possible(multi_item_tf_tensor)
+
+    assert e.value.error_code == ErrorCode.Name(INVALID_PARAMETER_VALUE)
+
+
+def test_convert_metric_value_to_float():
+    tf_tensor_val = tf.random.uniform([])
```

Comment:
```suggestion
    tf_tensor_val = tf.random.uniform([])  # pytlin: disable=no-value-for-parameter
```

Let's add a `# pytlin: disable=no-value-for-parameter` here to ignore the pylint error below.